Tests/L10nLintFrameworkTests/Resources/Fixtures/Localizables6/Base.lproj/Localizable.strings

```diff
@@ -0,0 +1,8 @@
+// MARK: KeyValueExtraSpaceRule
```

Comment:
`KeyValueExtraSpaceRule` is not suitable.
Please change to MixedChineseRule.

Reply:
Changed it.